students/charlie_england/session01/lesson1.py

```diff
@@ -0,0 +1,7 @@
+def sleep_in(weekday, vacation):
+    if weekday and not vacation:
+        return False
+    if not weekday:
```

Comment:
if an "if" and "else" both do the same thing -- do you need it at all?

Reply:
Yep! That was a silly mistake.